README.md

```diff
@@ -160,46 +167,22 @@ A number of options is available both for Composer plugin and a config loader. C
 },
 ```
 
-In the above `output-directory` points to where configs will be copied to. The path is relative to where
-`composer.json` is. The option is read for the root package, which is typically an application.
-Default is "/config/packages".
-
-If you change output directory, don't forget to adjust configs path when creating an instance of `Config`. Usually
-that is `index.php`:
-
-```php
-$config = new \Yiisoft\Config\Config(
-    dirname(__DIR__),
-    'config/packages', // Configs path.
-);
-
-$webConfig = $config->get('web');
-```
-
-`source-directory` points to where to read configs from for the package the option is specified for. The option is
-read for all packages. The value is a path relative to where package `composer.json` is. Default value is empty string.
-
-## Recursive merge of arrays 
+The `source-directory` specifies where to read the configs for the package for which the option is specified.
+The option is available for reading for all packages, including the root package, which is typically an application.
```

Comment:
It is currently set twice for the root package if it's an application:

1. In `composer.json`.
2. In `index.php` where we set path for `Config`.